### PR TITLE
[Example] Text classification with Tensorflow-Lite

### PR DESCRIPTION
**Pull request by gichan-jang**
Text classification example with Tensorflow-Lite.
When the user types a movie review,  pretrained model predicts if a paragraph's sentiment is positive or negative. 

TF-lite model can be downloaded here : https://www.tensorflow.org/lite/models/text_classification/overview

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ * ]Passed [ ]Failed []Skipped
2. Run test: [ * ]Passed [ ]Failed []Skipped

### CODE REVIEW

File: native/example_text_classification/meson.build [hunk 1]
```diff
@@ -0,0 +1,6 @@
+nnstreamer_example_text_classification_tflite = executable('nnstreamer_example_text_classification_tflite',
+  'nnstreamer_example_text_classification_tflite.c',
+  dependencies: [glib_dep, gst_dep, gst_base_dep, gst_app_dep, nns_dep, libdl_dep],
```
Comment (jaeyun-jung) [inline native/example_text_classification/meson.build]: gst_base_dep, nns_dep, libdl_dep  maybe unnecessary dependencies.

Comment (gichan-jang) [reply native/example_text_classification/meson.build]: As you mentioned, gst_base_dep, nns_dep and libdl_dep are removed.
Thank you for your comment.

**Comment by helloahn** [comment]
LGTM

**Comment by myungjoo** [comment]
Please update the get_model script to fetch the required files with
https://github.com/nnsuite/nnstreamer-example/blob/master/bash_script/example_models/get-model.sh

And then, if ```g_critical ("Failed to get model files.");``` happens, print out "how to download and locate the files with that updated get-models.sh".

**Comment by gichan-jang** [comment]
@helloahn 
> please upload the model file and the resources at https://github.com/nnsuite/testcases :)

The model file and the resources are uploaded at
https://github.com/nnsuite/testcases/tree/master/DeepLearningModels/tensorflow-lite/text_classification_tflite 

@myungjoo 
> Please update the get_model script to fetch the required files with
> https://github.com/nnsuite/nnstreamer-example/blob/master/bash_script/example_models/get-model.sh
> 
> And then, if `g_critical ("Failed to get model files.");` happens, print out "how to download and locate the files with that updated get-models.sh".

Bash script is updated and download guide is added when loading model files is failed.

**Pull request merged**